### DISCUSSION TITLE
Deutsche Übersetzung erhältlich? / German translation available?

### DISCUSSION THREAD

**Discussion by MdA193** [dominikamann/oekofen-pellematic-compact#48] [General]
Hallo Dominik Aman,
vielen Dank für die Oekofen-Integration im Home Assistant. Ist es möglich, eine Deutsche Ansicht der Entitäten zu integrieren?  /

Thank you for the Oekofen integration in the Home Assistant. Is it possible to integrate a German view of the entities?

**Comment by dominikamann** [comment]
Hi @MdA193,

das ist bestimmt möglich über https://developers.home-assistant.io/docs/internationalization/core/ und die Übersetzungen müssten dann entsprechend hier gepflegt werden https://github.com/dominikamann/oekofen-pellematic-compact/tree/main/custom_components/oekofen_pellematic_compact/translations. 

Aktuell habe ich jedoch nicht die Zeit das einzupflegen und entsprechend umzusetzen. Gerne darf am Code mitgearbeitet werden. Über einen PR könnte ich dann die Änderungen mit aufnehmen.

Alternativ können in HA die Namen der Entitäten manual bearbeitet werden. So ist es ebenfalls möglich mit etwas Aufwand die deutschen Übersetzungen zu setzen.

Viele Grüße,
Dominik

**Comment by MdA193** [comment]
Hallo,

vielen Dank für die Infos. Ich bin jedoch Anfänger in Sachen Home Automatisierung und bin auf den Home Assistant nur gestoßen, da ich nach einer Integration der Pellematic gesucht habe. Falls ich mal soweit bin, dass ich auch Code ändern könnte, würde ich natürlich gerne unterstützen. Aber ich glaube nicht, dass ich sowas hinbekommen würde.

Dann versuche ich es mal mit dem manuellen umbenennen.

VG

**Reply by dominikamann** [reply]
Grundsätzlich ein valider Punkt das einzuführen. Ich behalte es im Hinterkopf und eventuell finde ich mal Zeit dafür. 

Manuell gehts so: https://www.home-assistant.io/docs/configuration/customizing-devices/

Grüße

**Comment by MdA193** [comment]
Hallo Herr Amann,vielen Dank für den Link, schwer ist es wohl nicht ;-)Mein Problem ist nur, dass ich nicht alle Sensoren korrekt zuordnen kann. Da gibt es ein paar, auf die ich mir noch keinen Reim draus machen kann. Vielleicht schreib ich auch mal Oekofen direkt an für eine Beschreibung und korrekte Übersetzung, die ich dann einpflegen kann. VG und vielen Dank Am 20.03.2024 um 23:20 schrieb Dominik Amann ***@***.***>:﻿
Grundsätzlich ein valider Punkt das einzuführen. Ich behalte es im Hinterkopf und eventuell finde ich mal Zeit dafür.
Manuell gehts so: https://www.home-assistant.io/docs/configuration/customizing-devices/
Grüße
